test/createAzureResource.test.ts

```diff
@@ -120,6 +121,16 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
         assert.equal(createdApp.state, 'Running', `Function App state should be 'Running' rather than ${createdApp.state}.`);
     });
 
+    test('Add new setting', async () => {
+        const createdApp: WebSiteManagementModels.Site = await webSiteClient.webApps.get(resourceName1, resourceName1);
+        assert.ok(createdApp, `Function App ${resourceName1} not found`);
+        ext.ui = new TestUserInput([resourceName1, applicationSettings[0], applicationSettings[1]]);
+        await vscode.commands.executeCommand('azureFunctions.appSettings.add');
+        const listapplicationSettings: WebSiteManagementModels.StringDictionary = await webSiteClient.webApps.listApplicationSettings(resourceName1, resourceName1);
+        const applicationSettingsValue: string | undefined = applicationsettings(listapplicationSettings.properties, applicationSettings[0]);
+        assert.equal(applicationSettingsValue, applicationSettings[1], `The expected setting value should be "${applicationSettings[1]}" rather than "${applicationSettingsValue}"`);
```

Comment:
You'll have to verify that appSettings.properties isn't undefined, but then you can just check `appSettings.properties.applicationSettings[0] === applicationSettings[1]`